### ISSUE TITLE
3/16 + Test Database Connectivity + 1

### ISSUE THREAD

**Issue by anazariz** [lzim/teampsd#1185]
**1. Add description:** 

Please test end-user database connectivity by using the workbook in our team's folder / quant folder / ash work area / LSV. Open the (only) workbook in that folder and click on the 5 buttons in it. You should get either an error, which represents the issue that our end-users potentially have (with missing database privileges) or an OK, which represents folks in our team who have database access privileges, which is the desirable state for our end-users as well.
 
**Please note that actual folder and file names have not been used here.** and loose references assume that you have infrastructure knowledge.   

<< Paste screenshots here>>
 
**2. Click on the right:**  
i. **Pipelines** - prioritize in a TeamPSD (& Workgroup) pipeline  
ii. **Assignee** - assign to an individual team member producing work for the issue  
iii. **Label**- assign workgroup (lead) responsible for tracking the task  
iv. **Projects** - assign to domain-specific TeamPSD tracker (issues, features, document, manuscript, team)  
v. **Milestones** - select monthly sprint milestone within the workgroup   
vi. **Estimate** - Estimate task in complexity points   
vii. **Epics** - assign to team-wide epic and TeamPSD master plan project  
viii. **Releases** - assign to key team release grouped with other relevant issues  

**3. Once you hit "Submit new issue**, underneath the first comment:   
viiii. **Add dependency** - assess for and add dependencies *blocking* & *blocked by* this issue

**Comment by anazariz** [comment]
For some reason this issue was repeated 3 times after a screen freeze. Closing this one.
